library/src/main/java/ch/decent/sdk/utils/ElGamal.kt

```diff
@@ -7,7 +7,7 @@ import java.math.BigInteger
 object ElGamal {
 
   private val modulus = BigInteger("11760620558671662461946567396662025495126946227619472274601251081547302009186313201119191293557856181195016058359990840577430081932807832465057884143546419")
-  private val generator = 3.toBigInteger()
+  @Suppress("MagicNumber") private val generator = 3.toBigInteger()
```

Comment:
This Suppress rule shows up quite a bit. Why not just set the lint check to ignore this rule?

Reply:
it is used 12 times, further refactor could lower it to half
magic numbers should be avoided, there are exclusions for
`    ignoreNumbers: '-1,0,1,2,8,10,0xFF' `
in some cases specific cases you want suppress this check
i do not see a reason to disable this check